android/build.gradle

```diff
@@ -7,7 +7,7 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:7.2.0'
+        classpath 'com.android.tools.build:gradle:7.2.2'
```

Comment:
any reason we have updated this?

Reply:
Test cases was not working for 7.2.0 so upgraded the gradle version 7.2.2